app/controllers/concerns/membership_actions.rb

```diff
@@ -90,7 +90,7 @@ def access_levels # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
   def available_users
     # Remove current user from available users as a user cannot add themselves
     @available_users = User.where.not(id: Member.where(
```

Comment:
Can we change this to `User.where.not(id: Member.select(:user_id).where(namespace: @namespace)).where.not(id: current_user.id)`

Reply:
Sure! Updated in 0379505201abd0d8b0a317f46c1819e1c82f5c3e